modules/pipeline/aop/plugins/pipeline/apitest_report/plugin.go

```diff
@@ -29,22 +31,18 @@ const (
 	version2          = "2.0"
 )
 
-type Plugin struct {
-	aoptypes.PipelineBaseTunePoint
-}
-
-func (p *Plugin) Name() string {
-	return "apitest_report"
-}
-
 type ApiReportMeta struct {
 	ApiTotalNum   int `json:"apiTotalNum"`
 	ApiSuccessNum int `json:"apiSuccessNum"`
 	ApiFailedNum  int `json:"apiFailedNum"`
 	ApiNotExecNum int `json:"apiNotExecNum"`
 }
 
-func (p *Plugin) Handle(ctx *aoptypes.TuneContext) error {
+func New() *provider { return &provider{} }
```

Comment:
No need exported func `New()`.
`aop.NewProviderNameByPluginName(&provider{})` is enough.